locales/ru.json

```diff
@@ -3,40 +3,40 @@
+        "carddesc2": "Iris был разработан с нуля, чтобы быть гибким и легко обновляемым. Попрощайтесь с неделями ожидания поддержки шейдеров в новых версиях Minecraft!",
         "card3": "Бесплатный и открытый",
-        "carddesc3": "Iris - это мод с открытым исходным кодом, в который может внести свой вклад любой желающий. Это позволяет Iris совершенствоваться, облегчая сотрудничество.",
+        "carddesc3": "Iris - это open source мод, в который может внести свой вклад любой желающий. Это позволяет Iris использовать сотрудничество с комьюнити для разработки.",
         "heading2": "Возможности",
         "subheading2": "Что может предложить Iris",
-        "image1": "Паритет с OptiFine",
-        "imagedesc1": "Iris стремится иметь полную поддержку существовавших и существующих пакетов шейдеров OptiFine, и число шейдеров OptiFine, поддерживаемых в Iris стабильно и быстро растёт!<span class='text-span-2'><br>Скриншот от Niko</span>",
+        "image1": "Полная совместимость с OptiFine",
+        "imagedesc1": "Iris стремится иметь полную поддержку существовавших и существующих пакетов шейдеров OptiFine, и число шейдеров разработанных для OptiFine, поддерживаемых в Iris стабильно и быстро растёт!<span class='text-span-2'><br>Скриншот от Coder</span>",
         "image2": "Работает с Sodium",
-        "imagedesc2": "Iris совместим с модами оптимизации от CaffeineMC, такими как Sodium и Phosphor, обеспечивая потрясающую графику в кадрах, сравнимых с Vanilla, или даже лучше.<br><span class='text-span-2'>Скриншот от Guardian</span>",
+        "imagedesc2": "Iris совместим с модами оптимизации от CaffeineMC, такими как Sodium и Phosphor, обеспечивая потрясающую графику в кадрах, сравнимых с Vanilla, или даже лучше.<br><span class='text-span-2'>Скриншот от IMS</span>",
         "heading3": "Преимущества Iris",
-        "subheading3": "Новые методики рендеринга, которые ставят вас впереди игры",
+        "subheading3": "Новые методы рендеринга, которые меняют качество игры",
         "shadows": "Тени",
         "entities": "Сущности",
-        "screenshottitle1": "Оптимизированные тени",
-        "screenshotdesc1": "Iris использует технику, называемую Shadow Frustum Culling, чтобы уменьшить воздействие одной из самых больших проблем шейдеров - теней. Используя Shadow Frustum Culling, вы сможете достичь более высокой частоты кадров, не жертвуя качеством объёмного освещения или теней.<br><span class='text-span-2'>Скриншот от FoundationGames</span>",
+        "screenshottitle1": "Оптимизированный ренедринг теней",
+        "screenshotdesc1": "Iris использует технику, называемую Shadow Frustum Culling, чтобы уменьшить влияние на производительность одной из самых тяжелых частей шейдеров - теней. Используя Shadow Frustum Culling, вы сможете достичь гораздо более высокий FPS, не жертвуя качеством объёмного освещения или теней.<br><span class='text-span-2'>Скриншот от FoundationGames</span>",
```

Comment:
There are 3 problems in this line at once:

1. "_to reduce the overhead_" translates as "_уменьшить накладные расходы_", however, this is an extremely unpopular phrase, mainly used among programmers or accountants. I am sure that 99% of Russian-speaking readers will not understand it, so in the first version of the translation I localized it as "_уменьшить воздействие [на производительность]_" ("_reduce the impact [on performance]_"). The option proposed by enagimintas is also appropriate, but I would like to discuss this phrase as well. @thedocruby, if you think it's necessary to use direct translation, let us know.
2. The direct translation of "_one of the biggest performance hogs of a shader_" cannot be used because of the figurative expression, so this phrase also requires localization. I think that the original variant was more successful because there was a stronger semantic connection between this and the previous phrases (phrase "_воздействие проблемы_" = "_impact of the problem_" remains logical even without context; "_влияние части_" = "_impact of the part_" without context sounds weird).
3. "_достичь гораздо более высокий FPS_" ... The connection of endings between "_достичь_" and "_высокий_" is broken here and I will give a link to [the table of declensions of adjectives by case](https://1.bp.blogspot.com/-Mex0fPcYmdU/XVlJCvj5nrI/AAAAAAAAa0k/Fppl-NvZvlARc5uaglZewTj6EByptaZLwCLcBGAs/s1600/hello_html_3d82f500.jpg) in Russian. I also see no reason to leave the English abbreviation "_FPS_" untranslated when there is its direct equivalent in Russian – "_частота кадров_" - and when it can be used in context.

<details>
 <summary>Исходный текст</summary>

В этой строчке есть сразу 3 проблемы:

1. "_to decrease the overhead_" переводится как "_уменьшить накладные расходы_", однако это крайте непопулярная фраза, в основном используемая среди программистов или бухгалтеров. Я уверен, что 99% русскоязычных читателей не поймут её, поэтому в первичном варианте перевода локализовал её как "_уменьшить воздействие [на производительность]_". Предлагаемый enagimintas вариант тоже уместен, однако мне хотелось бы обсудить также эту фразу. @thedocruby , если вы считаете, что необходимо использовать прямой перевод, дайте знать.
2. Прямой перевод "_one of the biggest performance hogs of a shader_" использовать нельзя из-за образного выражения, поэтому эта фраза также требует локализации. Я считаю, что исходный вариант был более удачным из-за того, что в нём прослеживалась более сильная семантическая связь между этой и предыдущей фразами (фраза "_воздействие проблемы_" остаётся логичной даже без контекста; "_влияние части_" без контекста звучит странно).
3. "_достичь гораздо более высокий FPS_"... Я лишь скажу, что здесь нарушена падежная связь между "_достичь_" и "_высокий_" и дам ссылку на [таблицу склонений прилагательных по падежам](https://1.bp.blogspot.com/-Mex0fPcYmdU/XVlJCvj5nrI/AAAAAAAAa0k/Fppl-NvZvlARc5uaglZewTj6EByptaZLwCLcBGAs/s1600/hello_html_3d82f500.jpg) в русском языке (если эта информация до сих пор не закрепилась в голове). Также не вижу смысла оставлять непереведённой английскую аббревиатуру "_FPS_", когда есть её прямой эквивалент в русском языке – "_частота кадров_", – и когда его можно использовать в контексте.</details>

Reply:
I agree that we should optimize for readability, not technical accuracy.